solidity/contracts/isms/ccip-read/AbstractCcipReadIsm.sol

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
I assume you were planning on it, but a bit more of a README to show how folks should implement this contract would be nice